admin_guide/disabling_features.adoc

```diff
@@ -77,17 +77,9 @@ kubernetesMasterConfig:
 
 . Restart the {product-title} master service to apply the changes.
 +
-ifdef::openshift-enterprise[]
-----
-# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
-endif::[]
-ifdef::openshift-origin[]
+# systemctl master-restart api master-restart controllers
```

Comment:
@mburke5678 - I am guessing that this is correct? That it is the same command now for both origin and enterprise and that is why the ifdefs are removed?

Reply:
@vikram-redhat Yes. For controller (master), API, and etcd, the restart commands are the same for enterprise and origin. But the node restart commands are still different for each.
See [Scott's comment above](https://github.com/openshift/openshift-docs/pull/10293#issuecomment-399206510).